kotlin-reference-server/src/main/kotlin/org/stellar/reference/plugins/Sep24TestRoute.kt

```diff
@@ -40,19 +39,21 @@ fun Route.testSep24(
       }
 
       val transaction = sep24.getTransaction(transactionId)
-      val amountIn =
-        transaction.amountExpected?.amount?.toBigDecimal()
-          ?: throw ClientException("Missing amountExpected.amount field")
-
+      var amountExpected = transaction.amountExpected?.amount?.toBigDecimal()
+      if (amountExpected == null) {
```

Comment:
I don't think it's used anywhere right now? Amount is always passed in tests

Reply:
This is to support the use case when the `amount` is not specified in POST transactions. The test case will be added in `Sep24End2EndTests.kt`.

Reply:
Can you remove it from current iteration if the test is not there yet

Reply:
I think we should handle the case when the amount expected is not specified because the `amount` field is optional in SEP-24. https://github.com/stellar/stellar-protocol/blob/master/ecosystem/sep-0024.md#request-1

When the amount is not specified, we should give a number to simulate the interactive flow.

Reply:
I agree. But I don't think the way it's done now is good. Ideally there should be a function on ref server that you call from your test code. Just hardcoding it doesn't sounds like a good idea to me. 
Can you remove this change because it's out of scope of this PR?